Resources/Prototypes/Floof/Traits/lewd.yml

```diff
@@ -57,3 +61,29 @@
           reagents:
           - ReagentId: NaturalLubricant
             Quantity: 25
+
+- type: trait
+  id: ProductiveCumProducer
+  category: Physical
+  points: -2
+  requirements:
+    - !type:CharacterJobRequirement
+      inverted: true
+      jobs:
+        - Borg
+        - MedicalBorg
+    - !type:CharacterTraitRequirement
+      inverted: true
+      traits:
+        - CumProducer
+  components:
+    - type: ProductiveCumProducer
+      solutionname: "productivePenis"
+    - type: SolutionContainerManager
+      solutions:
+        productivePenis:
```

Comment:
Use the "penis" solutionname since you can't have both productiveCum and cum -producer traits.